spec/todos/projects/index_spec.rb

```diff
@@ -91,4 +91,63 @@
     it { is_expected.to have_http_status(:ok) }
     it { expect(response.body).to eq_json([project]) }
   end
+
+  context 'with pagination' do
+    before do
+      create_user_projects
+      current_user.update! admin: false, supervisor: false
+    end
+
+    context 'when page is unspecified' do
+      before do
+        get '/projects'
+      end
+
+      it { is_expected.to have_http_status(:ok) }
+      it { expect(JSON.parse(response.body).length).to eq(30) }
+      it { expect(response.headers['Total']).to be nil }
+      it { expect(response.headers['Per-Page']).to be nil }
+    end
+
+    context 'when first page' do
+      before do
+        get '/projects?page=1'
+      end
+
+      it { is_expected.to have_http_status(:ok) }
+      it { expect(JSON.parse(response.body).length).to eq(25) }
+      it { expect(response.headers['Total']).to eq('30') }
+      it { expect(response.headers['Per-Page']).to eq('25') }
```

Comment:
I think there was a test here for the links, what do you think about adding it?
an example of what this test should look like:
```ruby it 'expected links' do
      expect(response['Link'])
        .to eq(
          '<http://www.example.com/diamonds?page=2>; rel="last", ' \
          '<http://www.example.com/diamonds?page=2>; rel="next"'
        )
    end

Reply:
Aren't those links only for Rails views that we don't use?